internal/rbd/driver/driver.go

```diff
@@ -168,7 +182,7 @@ func (r *Driver) Run(conf *util.Config) {
 		if err != nil {
```

Comment:
@Madhu-1 @nixpanic 
I am thinking of removing this segment of code.
https://github.com/ceph/ceph-csi/blob/650c522ce016bcff86d590006d120f7b829fb93d/internal/rbd/driver/driver.go#L166-L176

changes to controllerserver and nodeserver standalone if statements are not propagated to this part from a long time (see krbd feature and replication sever), 
and I don't think this section is necessary. 

The complexity of this function will go down and it'll be much clearer.
wdyt?

Reply:
sounds good to remove it not required and make its less complex